index.html

```diff
@@ -39,5 +40,144 @@ <h2>LET'S CONNECT</h2>
         </ul>
         
     </section>
-</body>
-</html>
\ No newline at end of file
+    <section class="project-container">
```

Comment:
it would be better if you add some margin to divide between the sections as in the Figma Desgin 
![image](https://user-images.githubusercontent.com/61790587/166077954-eaee2a9f-1f07-4dea-ba8c-3e4a13d99482.png)